src/mon/PGMap.cc

```diff
@@ -3778,7 +3782,8 @@ int process_pg_map_command(
       } else {
         int filter = pg_string_state(state_str);
         if (filter < 0) {
-          *ss << "'" << state_str << "' is not a valid pg state";
+          *ss << "'" << state_str << "' is not a valid pg state,"
+              << " available choices: " << pg_state_string(-1);
```

Comment:
This is already handled by a ceph python binary:
```
$ bin/ceph -c ceph.conf pg ls somethingawful
somethingawful not valid:  somethingawful not in active|clean|down|scrubbing|degraded|inconsistent|peering|repair|recovering|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized|activating|peered                                                                    
Invalid command:  unused arguments: [u'somethingawful']                                                 
pg ls {<int>} {active|clean|down|scrubbing|degraded|inconsistent|peering|repair|recovering|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized|activating|peered [active|clean|down|scrubbing|degraded|inconsistent|peering|repair|recovering|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized|activating|peered...]} :  list pg with specific pool, osd, state                                                                  
Error EINVAL: invalid command                                                                           
```

What's the point of redoing it here?